src/implementations/vanilla/components/rating-field/_rating-field.scss

```diff
@@ -6,37 +6,56 @@
  */
 
 @use 'sass:map';
-@use '@ecl/theme-dev/theme';
 @use '@ecl/vanilla-layout-grid/mixins/breakpoints';
 
+// Exposed variables
+$theme: null !default;
+$rating-field: null !default;
+
 .ecl-rating-field {
```

Comment:
Not sure if this is just for this element, but when tere is no helper text, the stars are very close to the title
![image](https://github.com/ec-europa/europa-component-library/assets/25579221/1c714b45-689e-4504-b070-3b99c1c5cad6)

Reply:
oh ,wow...this was coming from the range component..